src/services/PurchaseOrderService.ts

```diff
@@ -0,0 +1,22 @@
+import api from '@/api';
+
+const fetchPurchaseOrders = async (payload: any): Promise <any>  => {
+  return api({
+   // TODO: We can replace this with any API
+    url: "/solr-query", 
+    method: "POST",
+    data: payload
+  });
+}
+
+const fetchPODetails = async (query: any): Promise <any> => {
```

Comment:
Unwanted method declaration for this PR, we can remove this.

Reply:
Removed Sir.